help/en/html/apps/DecoderPro/Ident.shtml

```diff
@@ -1,179 +1,139 @@
+        web page". We'll find a way to deal with anything that can be described in terms of CV
+        values.</li>
+
+        <li>As new types of decoders are produced, make sure to publish the specific information
+        needed to identify the decoder. This might be as simple as putting the identification
+        values on a web page for the decoder. (Since the values might change, putting them in an
+        manual is of limited use)</li>
+      </ul>
+      Beyond that:
       <ul>
-        <li>The most value for the customer would come if a
-        <em>specific</em> model can be identified. An additional CV
-        for this would be ideal, but any method of narrowing down
-        to a model 2XYZ123AB decoder would be ideal.</li>
-
-        <li>If concerned about separating "firmware update version"
-        from "feature set", consider using separate CVs for that,
-        and perhaps not even making the "firmware update"
-        information available to users.</li>
-
-        <li>An easy way to handle older decoders when moving to a
-        new method is to identify a specific CV7 value that hasn't
-        been used, and define that as a flag that other CVs should
+        <li>The most value for the customer would come if a <em>specific</em> model can be
+        identified. An additional CV for this would be ideal, but any method of narrowing down to a
+        model 2XYZ123AB decoder would be ideal.</li>
+
+        <li>If concerned about separating "firmware update version" from "feature set", consider
+        using separate CVs for that, and perhaps not even making the "firmware update" information
+        available to users.</li>
+
+        <li>An easy way to handle older decoders when moving to a new method is to identify a
+        specific CV7 value that hasn't been used, and define that as a flag that other CVs should
         be consulted for more information.</li>
 
-        <li>When new features are added to a decoder, consider
-        calling it a new model. Having many different versions of
-        the XYZ123 decoder, some with BEMF, some without, some with
-        special lighting options, some without, etc, confuses both
-        the customer and the people trying to support customers
-        with software like DecoderPro. And why not get credit for
-        the new features from people who'll ask for the new
-        model?</li>
-      </ul><!--#include virtual="/Footer.shtml" -->
-    </div><!-- closes #mainContent-->
-  </div><!-- closes #mBody-->
+        <li>When new features are added to a decoder, consider calling it a new model. Having many
+        different versions of the XYZ123 decoder, some with BEMF, some without, some with special
+        lighting options, some without, etc, confuses both the customer and the people trying to
+        support customers with software like DecoderPro. And why not get credit for the new
+        features from people who'll ask for the new model?</li>
+      </ul>
```

Comment:
I take back what I said about needing blank lines:  cf. old line 175 which becomes new lines 131 and 132 (unless you did this manually).  This is what should happen, IMHO.

OTOH, old line 175 has a comment on the same line as the /div which gets split to new lines 133 and 134, which is acceptable.